configure.ac

```diff
@@ -5797,6 +5805,12 @@ then
     AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_DES_ECB -DHAVE_EX_DATA -DWOLFSSL_KEY_GEN"
 fi
 
+if test "$ENABLED_MEMCACHED" = "yes"
+then
+    AM_CFLAGS="$AM_CFLAGS -DDEBUG_WOLFSSL_VERBOSE -DWOLFSSL_SESSION_ID_CTX"
```

Comment:
Is `DEBUG_WOLFSSL_VERBOSE` really always required?

Reply:
Looks like it was added originally for ERR_print_errors_fp, before I made a dedicated memcached flag HAVE_MEMCACHED.  Now that I have that flag, I will just gate the function ERR_print_errors_fp behind it so memcached doesn't pull in other verbose logging functions.